pages/linux/dnf-group.md

```diff
@@ -0,0 +1,24 @@
+# dnf group
+
+> Manage groups of related DNF packages on Fedora-based systems.
+> More information: <https://manned.org/man/dnf-group>.
+
+- List DNF groups, showing installed and uninstalled status in a table:
+
+`dnf group list`
+
+- Show DNF group info, including repository and optional packages:
+
+`dnf group info {{groupname}}`
+
+- Install DNF group:
+
+`dnf group install {{groupname}}`
+
+- Remove DNF group:
+
+`dnf group remove {{groupname}}`
+
+- Upgrade DNF group:
+
+`dnf group upgrade{{groupname}}`
```

Comment:
Missed a space here?

Reply:
![image](https://github.com/user-attachments/assets/1db28282-b8a5-45ec-a535-0d572eecd799)

@cyqsimon There's already a space there (checked in this PR's Files Changed tab), please recheck - the command is showing with space only:

`dnf group upgrade {{groupname}}`